src/app/view/components/_common/searchable-dropdown/component/searchable-dropdown.component.html

```diff
@@ -17,16 +17,18 @@
         [value]="data"
         class="searchable-dropdown__data-list">
         <ng-template let-item pTemplate="item">
-            <span [class.selected]="item[labelPropertyName] === valueString" (click)="handleClick(item)">{{ getLabel(item) }}</span>
+            <span [class.selected]="item[labelPropertyName] === valueString" (click)="handleClick(item)">{{ getItemLabel(item) }}</span>
```

Comment:
Can you create a property to the class and assign the value? if you put a `console.log` in the `getItemLabel` you'll see it will be called a bunch of times.